src/include/ureport.h

```diff
@@ -94,9 +92,8 @@ ureport_server_config_destroy(struct ureport_server_config *config);
  *
  * @param config a server configuration to be populated
  */
-#define ureport_server_config_load libreport_ureport_server_config_load
 void
-ureport_server_config_load(struct ureport_server_config *config,
+libreport_ureport_server_config_load(struct ureport_server_config *config,
                            map_string_t *settings);
```

Comment:
The broken alignment will be an issue everywhere. Can be addressed later ad-hoc.